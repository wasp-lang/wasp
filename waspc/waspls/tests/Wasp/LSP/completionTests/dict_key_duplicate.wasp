//! test/completion
app todoApp {
  wasp: {
    version: "^0.10.5", 
  },
  title: "todo!",
}

// we suggest all keys of a dictionary, even when some are already used  
route MainRoute { path: "/", | }
                             ^
page MainPage {
  component: import { MainPage } from "@client/MainPage.jsx",
}
