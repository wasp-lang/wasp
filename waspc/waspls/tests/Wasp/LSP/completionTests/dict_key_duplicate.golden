Completion items:
  label={"path"} kind={CiField} detail={":: string"} insertText={"path: "}
  label={"to"} kind={CiField} detail={":: page (declaration type)"} insertText={"to: "}
